### DISCUSSION TITLE
[Enhancement] [input-number] el-input-number的小数位是否展示应该依赖户主动的输入

### DISCUSSION THREAD

**Discussion by dang0705** [element-plus/element-plus#15286] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `input-number`

## Description

一旦设置了属性precision, 无论用户是否输入了小数, 都强制会显示小数,如果precision 我传入了一个比较大的值,比如8, 展示上就会比较奇怪,而且这个属性应该是一个最大支持的小数位而非强制显示小数位的逻辑

### Additional comments

_(empty)_

**Comment by chenxch** [comment]
强制显示应该是大家对于精度属性设计上的共识吧，直接明了的表示这个字段是一个精度多少的数字。
